packages/node_modules/pouchdb-selector-core/src/utils.js

```diff
@@ -4,6 +4,9 @@ import { clone } from 'pouchdb-utils';
 // can be "deep" due to dot notation
 function getFieldFromDoc(doc, parsedField) {
   var value = doc;
+  if (!value) {
+    return
```

Comment:
It makes more sense to me to define the return type explicitly:
```suggestion
    return undefined;
```